mooncake-common/common.cmake

```diff
@@ -46,6 +46,16 @@ option(USE_HTTP "option for enable http as metadata server" ON)
 option(WITH_RUST_EXAMPLE "build the Rust interface and sample code for the transfer engine" OFF)
 option(WITH_METRICS "enable metrics and metrics reporting thread" ON)
 
+
+option(USE_LRU_MASTER "option for using LRU in master service" OFF)
+set(LRU_MAX_CAPACITY 1000)
```

Comment:
It would be great to support this as a startup flag in the master process (using gflags).